tilequeue/store.py

```diff
@@ -74,12 +108,17 @@ def write_tile(self, tile_data, coord, format, layer):
         key_name = s3_tile_key(
             self.date_prefix, self.path, layer, coord, format.extension)
         key = self.bucket.new_key(key_name)
-        key.set_contents_from_string(
-            tile_data,
-            headers={'Content-Type': format.mimetype},
-            policy='public-read',
-            reduced_redundancy=self.reduced_redundancy,
-        )
+
+        @_backoff_and_retry(Exception)
```

Comment:
It'd be nice if there was a logger we could pass in so we know when things retry rather than stuff just silently taking forever.